### DISCUSSION TITLE
[Enhancement] Expense data refinement

### DISCUSSION THREAD

**Discussion by logeshpaul** [ananthakumaran/paisa#7] [Ideas]
![image](https://user-images.githubusercontent.com/41541/211132633-b08fd8c1-d339-47eb-88d1-31652c476992.png)

Can introduce Yearly, Monthly, Week, date range filters.

**Comment by ananthakumaran** [comment]
The expense page has been split into monthly and yearly. I am not convinced about weekly/custom date range as of now, as it's hard to visualize and the real use cases might be low.

https://paisa-demo.ananthakumaran.in/expense/yearly
